invenio_stats/contrib/record_view/record_view_v1.json

```diff
@@ -1,5 +1,5 @@
 {
-  "template": "events-recordview-*",
+  "template": "events-stats_record_view-*",
```

Comment:
Could you rename the alias to match the template name too, i.e. `events-stats_record_view`